content/en/docs/contribute/generate-ref-docs/kubectl.md

```diff
@@ -48,64 +48,80 @@ information, see
 
 {{% capture steps %}}
 
-## Getting three repositories
+## Setting up the local repositories
 
-If you don't already have the kubernetes/kubernetes repository, get it now:
+Create a local workspace and set your `GOPATH`.
 
 ```shell
-mkdir $GOPATH/src
-cd $GOPATH/src
-go get github.com/kubernetes/kubernetes
+mkdir -p $HOME/<workspace>
+
+export GOPATH=$HOME/<workspace>
 ```
 
-Determine the base directory of your clone of the
-[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repository.
-For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes/kubernetes.`
-The remaining steps refer to your base directory as `<k8s-base>`.
+Get a local clone of the following repositories:
+
+```shell
+go get -u github.com/spf13/pflag
+go get -u github.com/spf13/cobra
+go get -u gopkg.in/yaml.v2
+go get -u kubernetes-incubator/reference-docs
+```
 
 If you don't already have the kubernetes/website repository, get it now:
 
 ```shell
-mkdir $GOPATH/src
-cd $GOPATH/src
-go get github.com/kubernetes/website
+git clone https://github.com/<your-username>/website $GOPATH/src/github.com/<your-username>/website
 ```
 
-Determine the base directory of your clone of the
-[kubernetes/website](https://github.com/kubernetes/website) repository.
-For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes/website.`
-The remaining steps refer to your base directory as `<web-base>`.
+Get a clone of the kubernetes/kubernetes repository as k8s.io/kubernetes:
+
+```shell
+git clone https://github.com/kubernetes/kubernetes $GOPATH/src/k8s.io/kubernetes
```

Comment:
k/kubernetes is now switched to go modules. Is it still required to clone into $GOPATH?

Reply:
hello @ialidzhikov . Right, what do you suggest? There are various ways to setup the build for the kubectl reference. I would suggest the most straightforward and repeatable process.

Reply:
Starting in Go 1.13 (1 August, 2019), module mode will be the default for all development. The major feature of go modules is that they allow you to decouple from the GOPATH and develop outside of it. 
If possible, I would not git clone in GOPATH.

Reply:
OK. Is this the best doc which explains the migration to [go modules](https://github.com/golang/go/wiki/Modules)? Typically, the docs reference build is required for what I would call major releases, 1.13, 1.14 with the main k8s build components being kubectl and now kustomize. These two components are or will be in repos outside of the main k8s/k8s repo?